nodemon.json

```diff
@@ -0,0 +1,9 @@
+{
+  "watch": ["api/", "api-lib/", "scripts/"],
+  "delay": 3000,
+  "execMap": {
+    "ts": "ts-node --swc"
+  },
+  "events": { "start": "yarn typecheck:backend" },
```

Comment:
```suggestion
```

Running 

❯ yarn typecheck:backend
yarn run v1.22.19
$ tsc -p tsconfig-backend.json --noEmit
✨  Done in 12.36s.

takes 12 seconds on my machine... so could we just omit this start event and let errors bubble up if necessary to have faster start times? 

Or if not that , run it async from the yarn start command somehow?

Reply:
Actually, the slowness of this start only happened on the first run, and it seems to cache and start faster after that. 

I'm seeing starts of 12s locally, and reloads in 1-2 seconds.

Reply:
The initial lod is due to the frontend"'s initial compilation.